src/device/lib.rs

```diff
@@ -210,21 +190,65 @@ impl<R: Resources, T> BufferHandle<R, T> {
 }
 
 /// Raw (untyped) Buffer Handle
-pub type RawBufferHandle<R: Resources> = Handle<<R as Resources>::Buffer, BufferInfo>;
+#[derive(Copy, Clone, PartialEq, Debug)]
+pub struct RawBufferHandle<R: Resources> {
+    /// The name of the raw buffer.
+    pub name: <R as Resources>::Buffer,
```

Comment:
We can't have this `pub`, because it would allow 3rd parties to construct bogus handles. `info` can still be public, this would make `get_info()` irrelevant.